metagraph/src/graph/alignment/aligner_seeder_methods.cpp

```diff
@@ -166,7 +166,8 @@ void SuffixSeeder<BaseSeeder>::generate_seeds() {
     }
 
     const DBGSuccinct &dbg_succ = get_base_dbg_succ(&this->graph_);
-    assert(!dbg_succ.get_mask());
+    if (dbg_succ.get_mask())
+        logger->warn("Graph has a dummy k-mer mask. Seeds containing dummy k-mers will be missed.");
```

Comment:
Seems like it's better to have the mask missing? If so, can't you just ignore it?

Reply:
Technically I can, but this would overcomplicate the code because I would have to have some way to know whether an index stored in a seed represents a BOSS edge or a DBGSuccinct node.